### DISCUSSION TITLE
NextJS application not able to run

### DISCUSSION THREAD

**Discussion by Saiyog** [vercel/next.js#73588] [Help]
### Summary

I have created simple nextJS application with command **npx create-next-app@latest myfirstapp** and without doing any changes trying to run app with command **npm run dev**, but facing an issue, after running an command running state is stucked on **Ready in seconds** state. Please find attached Screenshot for more details
![nextjs SC](https://github.com/user-attachments/assets/56ec0a81-ed2a-4a10-834e-b240c516bcef)
![nextjs SC](https://github.com/user-attachments/assets/b90d2227-02ef-4c23-8a02-841c9ea4cb71)

**Comment by icyJoseph** [comment]
This might sound silly, but have you tried to open `localhost:3000`? the app doesn't autostart a browser window, like other frameworks might do. In fact it is lazy compiled, so pages are build as your app needs them.
